oxidauth-kernel/src/user_authorities/create_user_authority.rs

```diff
@@ -25,18 +22,26 @@ pub type CreateUserAuthorityService = Arc<
 #[derive(Debug, Serialize, Deserialize)]
 pub struct CreateUserAuthorityParams {
     pub user_id: Uuid,
-    pub strategy: AuthorityStrategy,
+    pub client_key: Uuid,
     pub params: JsonValue,
 }
 
-impl From<&CreateUserAuthorityParams> for FindAuthorityByStrategy {
+impl From<&CreateUserAuthorityParams> for FindAuthorityByClientKey {
     fn from(value: &CreateUserAuthorityParams) -> Self {
         Self {
-            strategy: value.strategy,
+            client_key: value.client_key,
         }
     }
 }
 
+// impl From<&CreateUserAuthorityParams> for FindAuthorityByStrategy {
+//     fn from(value: &CreateUserAuthorityParams) -> Self {
+//         Self {
+//             strategy: value.strategy,
+//         }
+//     }
+// }
```

Comment:
We should un-comment or delete

```suggestion
// impl From<&CreateUserAuthorityParams> for FindAuthorityByStrategy {
//     fn from(value: &CreateUserAuthorityParams) -> Self {
//         Self {
//             strategy: value.strategy,
//         }
//     }
// }
```